Utils.py

```diff
@@ -392,7 +395,8 @@ def find_class(self, module, name):
         # Options and Plando are unpickled by WebHost -> Generate
         if module == "worlds.generic" and name in {"PlandoItem", "PlandoConnection"}:
             return getattr(self.generic_properties_module, name)
-        if module.endswith("Options"):
+        # pep 8 specifies that modules should have "all-lowercase names" (options, not Options)
+        if module.endswith("ptions"):
```

Comment:
@Berserker66 i'm not sure when this is even used? Assuming this allows and is required for using Option classes from worlds/*/Options.py, this should be properly documented somewhere (world api?)

Reply:
I think the reason I ran into this is that I named my file `worlds/zillion/options.py` and something didn't work.
It was a while ago, so I don't remember for sure.

Reply:
AsyncTools uses it. Maybe we make it an api and have AsyncTools add this.

Reply:
does this mean you approve of the change for the time being?

Reply:
For clarity:

The issue is that people can't follow pep8, because this part of the system requires the uppercase `O` in `Options`.

This change allows people to follow pep8, by allowing `options` or `Options` in this part of the system.

Reply:
Yeah, that sums it up. I didn't know it was required to be Options.py either. So far we've been trying to keep naming consistent rather than pep8-ey, which is probably why it never came up.

What I was asking is *why* we even need to pickle/unpickle the options. Neither the save nor the multidata should require this, and async / AsyncTools can have a different trust barrier (a different pickler).

Reply:
I could imagine someone making a module called `captions.py` or `inceptions.py`, and something strange happens...

If you want, I could change it to:
`if module.endswith("Options") or module.endswith("options"):`

Reply:
changed to
`module.lower().endswith("options")`